src/js/view/controls/components/settings/menu.js

```diff
@@ -63,15 +63,15 @@ export function SettingsMenu(onVisibility, buttonVisibilityChanges, localization
                 break;
         }
         sourceEvent.stopPropagation();
-        if (/13|32|37|38|39|40/.test(sourceEvent.keyCode)) {
+        if (/13|27|32|37|38|39|40/.test(sourceEvent.keyCode)) {
```

Comment:
Why do you want to preventDefault on esc?

Reply:
Verified it doesn't prevent exiting fullscreen (event if menu open). I added it to prevent unintended scrolling when closing menu with escape.